[2024-06-18T14:22:10.114+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T14:22:10.138+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_ecomerc_pipeline_test_extract_v2.load_cvs_to_staging manual__2024-06-18T14:22:09.603534+00:00 [queued]>
[2024-06-18T14:22:10.145+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_ecomerc_pipeline_test_extract_v2.load_cvs_to_staging manual__2024-06-18T14:22:09.603534+00:00 [queued]>
[2024-06-18T14:22:10.146+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-18T14:22:10.156+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_cvs_to_staging> on 2024-06-18 14:22:09.603534+00:00
[2024-06-18T14:22:10.163+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_ecomerc_pipeline_test_extract_v2', 'load_cvs_to_staging', 'manual__2024-06-18T14:22:09.603534+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/ecom_dag.py', '--cfg-path', '/tmp/tmpdu20pwf1']
[2024-06-18T14:22:10.165+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=210) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-18T14:22:10.165+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask load_cvs_to_staging
[2024-06-18T14:22:10.166+0000] {standard_task_runner.py:63} INFO - Started process 212 to run task
[2024-06-18T14:22:10.204+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_ecomerc_pipeline_test_extract_v2.load_cvs_to_staging manual__2024-06-18T14:22:09.603534+00:00 [running]> on host 8ff8f672542d
[2024-06-18T14:22:10.392+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_ecomerc_pipeline_test_extract_v2' AIRFLOW_CTX_TASK_ID='load_cvs_to_staging' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T14:22:09.603534+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-18T14:22:09.603534+00:00'
[2024-06-18T14:22:10.394+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T14:22:10.405+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T14:22:10.406+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/etl/extract_load_staging.py", line 30, in load_to_staging
    cursor.execute(load_csv)
psycopg2.errors.UndefinedFile: could not open file "/opt/airflow/dags/customers.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2024-06-18T14:22:10.416+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_ecomerc_pipeline_test_extract_v2, task_id=load_cvs_to_staging, run_id=manual__2024-06-18T14:22:09.603534+00:00, execution_date=20240618T142209, start_date=20240618T142210, end_date=20240618T142210
[2024-06-18T14:22:10.424+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 7 for task load_cvs_to_staging (could not open file "/opt/airflow/dags/customers.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
; 212)
[2024-06-18T14:22:10.460+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-18T14:22:10.474+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T14:22:10.475+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
